README.md

```diff
@@ -1,4 +1,119 @@
+`"{nickname} is playing a song on the {musical_instrument}"`
+
+All dwarves like delicious food so
+`Dwarf` constructor should take an additional argument `favourite_dish` -  and stored it in the **protected** attribute.
+Also, it should have a method `eat_favourite_dish` that should print the following string:
+`"{nickname} is eating {favourite_dish}"`
+
+Finally, create four classes: `ElfRanger`, `Druid`, `DwarfWarrior` and `DwarfBlacksmith`.
+
+`ElfRanger` should be a child of `Elf`.
+Its constructor should take one additional parameter: 
+`bow_level` - an integer that shows the quality of the bow.
+Constructor should store it in the protected attribute.
+
+`Druid` should be a child of `Elf`.
+Its constructor should take one additional parameter: 
+`favourite_spell` - the text of the favourite spell.
+
+`DwarfWarrior` should be a child of `Dwarf`.
+Its constructor should take one additional parameter: 
+`hummer_level` - an integer that shows the power of the hummer.
+
+`DwarfBlacksmith` should be a child of `Dwarf`.
+Its constructor should take one additional parameter: 
+`skill_level` - an integer that shows the level of blacksmith's skills.
+
+All of these classes should have implementations of `player_info` and 
+`get_rating` methods.
+
+`player_info` method should return:
+* `"Elf ranger {nickname}. {nickname} has bow of the {bow_level} level"` for `ElfRanger` instances
+* `"Druid {nickname}. {nickname} has a favourite spell: {favourite_spell}"` for `Druid` instances
+* `"Dwarf warrior {nickname}. {nickname} has a hummer of the {hummer_level} level"` for `DwarfWarrior` instances
+* `"Dwarf blacksmith {nickname} with skill of the {skill_level} level"` for `DwarfBlacksmith` instances
+
+
+`get_rating` method should return:
+* 3 * `bow_level` for `ElfRanger`
+* length of `favourite_spell` for `Druid`
+* `hummer_level` + 4 for `DwarfWarrior`
+* `skill_level` + 4 for `DwarfBlacksmith`
+
+```python
+ranger = ElfRanger(
+    nickname="Nardual Chaekian",
+    musical_instrument="flute",
+    bow_level=7
+)
+ranger.get_rating() == 21
+ranger.player_info() == "Elf ranger Nardual Chaekian. Nardual Chaekian has bow of the 7 level"
+ranger.play_elf_song()  # "Nardual Chaekian is playing a song on the flute" 
+```
+
+```python
+warrior = DwarfWarrior(
+    nickname="Thiddeal",
+    favourite_dish="French Fries",
+    hummer_level=7
+)
+ranger.get_rating() == 11
+ranger.player_info() == "Dwarf warrior Thiddeal. Thiddeal has a hummer of the 7 level"
+ranger.play_elf_song()  # "Thiddeal is eating French Fries" 
+```
+
+
+
+Write following functions:
+* `calculate_team_total_rating` - 
+it should take list of `Player`s and return the sum of the ratings for all team members
+```python
+team = [
+    Druid(nickname="Druid", musical_instrument="flute", favourite_spell="ABC"),
+    ElfRanger(nickname="Ranger", musical_instrument="trumpet", bow_level=33),
+]
+calculate_team_total_rating(team) == 99
```

Comment:
Please, add here an explanation - why 99 (and I believe, that here it is len("ABC") = 3, 3 * 33 = 99. So 102 in total.